kafkautil/formatters_test.go

```diff
@@ -0,0 +1,67 @@
+package kafkautil
+
+import (
+	"testing"
+
+	kafka "github.com/confluentinc/confluent-kafka-go/v2/kafka"
+	"github.com/spf13/viper"
+)
+
+func TestFmtMessageId(t *testing.T) {
+	type args struct {
+		offset kafka.Offset
+		delay  int64
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "Correct Format",
+			args: args{
+				offset: 0,
+				delay:  12345,
+			},
+			want: "0-12345",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := FmtMessageId(tt.args.offset, tt.args.delay); got != tt.want {
```

Comment:
great, while this is a very clean way to write it I just want to tip yopu on the google `cmp` library, it has some great shorthand functions to compare values and structs

Reply:
I will check it out!